akka-docs/rst/scala/code/docs/actor/SharedMutableStateDocSpec.scala

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package docs.actor
+
+class SharedMutableStateDocSpec {
+
+  //#mutable-state
+  import akka.actor.{ Actor, ActorRef }
+  import akka.pattern.ask
+  import akka.util.Timeout
+  import scala.concurrent.Future
+  import scala.concurrent.duration._
+  import scala.language.postfixOps
+  import scala.collection.mutable
+
+  case class Message(msg: String)
+
+  class EchoActor extends Actor {
+    def receive = {
+      case msg => sender() ! msg
+    }
+  }
+
+  class CleanUpActor extends Actor {
```

Comment:
I only wanted to point out that if the example would be too long (or verbose) for the docs pages, these 2 actors are so dummy that could be removed.  
I would keep them, so the examples can run correctly.